src/test/java/uk/gov/hmcts/juror/api/moj/report/AbstractReportTest.java

```diff
@@ -914,6 +934,146 @@ void positiveTypicalOwnersMustMatch() {
+                        .build(),
+                    "names", AbstractReportResponse.DataTypeValue.builder()
+                        .displayName("Names")
+                        .dataType(String.class.getSimpleName())
+                        .value("Someone Name")
+                        .build(),
+                    "court_room", AbstractReportResponse.DataTypeValue.builder()
+                        .displayName("Court Room")
+                        .dataType(String.class.getSimpleName())
+                        .value("COURT 3")
+                        .build(),
+                    "judge", AbstractReportResponse.DataTypeValue.builder()
+                        .displayName("Judge")
+                        .dataType(String.class.getSimpleName())
+                        .value("Judge Dredd")
+                        .build(),
+                    "court_name", AbstractReportResponse.DataTypeValue.builder()
+                        .displayName("Court Name")
+                        .dataType(String.class.getSimpleName())
+                        .value("Chester (415)")
+                        .build()
+                ));
+
+            verify(report, times(1))
+                .getTrial(TestConstants.VALID_TRIAL_NUMBER, trialRepository);
+            verify(request, times(2)).getTrialNumber();
+            verify(trial, times(2)).getCourtroom();
+            verify(trial, times(1)).getDescription();
+            verify(courtLocation, times(1)).getLocCode();
+            verify(trial, times(3)).getCourtLocation();
+        }
+
+        @Test
+        void negativeTrialNotFound() {
+            TrialRepository trialRepository = mock(TrialRepository.class);
+
+            AbstractReport<Object> report = createReport();
+
+            Assertions.assertThatExceptionOfType(MojException.NotFound.class).isThrownBy(() ->
+                report.getTrial(TestConstants.VALID_TRIAL_NUMBER, trialRepository));
+        }
+
+        @Test
+        void negativeInvalidAccess() {
+
+            Trial trial = mock(Trial.class);
+            AbstractReport<Object> report = createReport();
+
+            doReturn(trial).when(report).getTrial(any(), any());
+            securityUtilMockedStatic.when(SecurityUtil::isCourt).thenReturn(false);
+
+            report.isCourtUserOnly();
+            assertThat(report.authenticationConsumers).hasSize(1);
+            Consumer<StandardReportRequest> authenticationConsumer = report.authenticationConsumers.get(0);
+
+            MojException.Forbidden exception = assertThrows(
+                MojException.Forbidden.class,
+                () -> authenticationConsumer.accept(mock(StandardReportRequest.class)),
+                "Expected exception to be thrown when not court user");
+            assertThat(exception.getMessage()).isEqualTo("User not allowed to access this report");
+            assertThat(exception.getCause()).isNull();
+        }
+    }
```

Comment:
This test is not needed here should be one on each report test that restricts user type

Reply:
Removed unneeded test